tests/basic/reboot_test.go

```diff
@@ -253,3 +253,93 @@ var _ = Describe("{ReallocateSharedMount}", func() {
 		AfterEachTest(contexts, testrailID, runID)
 	})
 })
+
+var _ = Describe("{ClusterPxRestart}", func() {
+	var testrailID = 60042
+	// testrailID corresponds to: https://portworx.testrail.net/index.php?/cases/view/60042&group_by=cases:section_id&group_order=asc&display_deleted_cases=0&group_id=4120
```

Comment:
fix this url, compare the url in other tests

Reply:
done